extension/bg/js/background.js

```diff
@@ -91,6 +91,10 @@ async function recordFlashcard(lines, start, end, currentVideoTime, audioTrack)
         screenshotToSendBack = imageBase64;
     }
 
+    if(neededExpressions.has('filename')) {
+        expressionLookup['filename'] = videoFileName;
```

Comment:
Any reason this can't be `videoFile.name` instead? Then you can avoid having to pipe everything through CardCreator.

Reply:
I apparently just missed `videoFile` completely.